modules/gapi/src/backends/fluid/gfluidimgproc.cpp

```diff
@@ -1097,7 +1103,8 @@ GAPI_FLUID_KERNEL(GFluidFilter2D, cv::gapi::imgproc::GFilter2D, true)
 
         int k_rows = kernel.rows;
         int k_cols = kernel.cols;
-        const float *k = scratch.OutLine<float>(); // copy of kernel.data
+
+        float *k = scratch.OutLine<float>(); // copy of kernel.data
```

Comment:
Should const actually be removed here?

Reply:
Good point, thanks! I would restore `const` qualifier here.